src/app/js/fields/ontology/OntologyFieldList.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { SortableContainer } from 'react-sortable-hoc';
+import { TableBody, TableRow } from 'material-ui/Table';
+
+import OntologyField from './OntologyField';
+import { COVER_DATASET } from '../../../../common/cover';
+
+const OntologyFieldList = SortableContainer(({ items, filter }) => (
+    <TableBody displayRowCheckbox={false}>
+        {items
+            .filter(
+                field =>
```

Comment:
extract the filter function, so that it's unit testeable, and appears as a simple name here (better for readability). same for the other filter function below.